tests/installation/first_boot.pm

```diff
@@ -31,7 +37,7 @@ sub run() {
         send_key "ret";
     }
 
-    # Check for errors during first boot
+    # Prepare the KDE desktop for the first run
     my $err  = 0;
```

Comment:
it's not just KDE - it might be in your test, but first_boot.pm runs for a lot of tests and desktops